### PR TITLE
Specialization dropdown

### LINKED ISSUE #3: [3pt] Add Item
- Create a Form to add a new Item
- Autofill username
- When a item is successfully added, show it in Item page.

### PR DESCRIPTION
**Pull request by anita00001**
## In this PR, I have added the following:
- Removed Hard-coded Specialization Options
- Fetched Specialization from the API
- Dynamic Specialization Dropdown
- The Dropdown option of the Specialization on Create an Expert Page has all Specializations in the Specialization Table.

### CODE REVIEW

File: src/components/Specializations/SpecializationDropdown.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  return (
+    <select
+      name="specialization_id"
+      value={formData.specialization}
+      onChange={onChange}
+    >
+      <option value="">Select Specialization</option>
+      {specializations.map((specialization) => (
+        <option key={specialization.id} value={specialization.id}>{specialization.name}</option>
+      ))}
```
Comment (Mofuhidy) [inline src/components/Specializations/SpecializationDropdown.js:23]: In this part of code it shows me this error

Screenshot 2023-11-06 at 11 05 49 AM

**Review by Mofuhidy — CHANGES_REQUESTED** [review]
# Changes required ♻️  : 

While you did a great job, but there is something to check.
Also we need to add alert for adding the specialization.

File: src/components/Specializations/SpecializationDropdown.js (hunk 1 above)
Comment (anita00001) [reply src/components/Specializations/SpecializationDropdown.js:23]: Thank you for noticing the issue. We have fixed the issue in new commits.

**Review by Mofuhidy — APPROVED** [review]
### APPROVOED 👏👏👏👏
Great work refactoring the codes. At this point, there is nothing else to do other than its time to merge it. Happy coding !!!!



### Highlights
✔Accept Gitflow
✔ Accepted commit messages
✔️ Specializations list implemented.

**Pull request merged**